llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -375,7 +375,6 @@ AMDGPUTargetLowering::AMDGPUTargetLowering(const TargetMachine &TM,
   setTruncStoreAction(MVT::v16f64, MVT::v16bf16, Expand);
   setTruncStoreAction(MVT::v16f64, MVT::v16f16, Expand);
   setTruncStoreAction(MVT::v16i64, MVT::v16i16, Expand);
-  setTruncStoreAction(MVT::v16i64, MVT::v16i16, Expand);
```

Comment:
Added by https://reviews.llvm.org/D79960 . I wonder if it was meant to say `v16i32`. @rampitec